command/src/logging/mod.rs

```diff
@@ -13,5 +13,19 @@ pub mod display;
 #[macro_use]
 pub mod logs;
 
+use std::io::Error as IoError;
```

Comment:
You could remove it

Reply:
this alias is already used throughout Sōzu, but sure